website/docs/d/pi_key.html.markdown

```diff
@@ -8,7 +8,7 @@ description: |-
 ---
 
 # ibm_pi_key
-Retrieve information about the SSH key that is used for your Power Systems Virtual Server instance. The SSH key is used to access the instance after it is created. For more information, about [configuring your IBM i virtual machine (VM)](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-configuring-ibmi).
+Retrieve information about the SSH key that is used for your Power Systems Virtual Server instance. The SSH key is used to access the instance after it is created. For more information, about [configuring your IBM virtual machine (VM)](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-configuring-ibmi).
```

Comment:
The URL is still pointing to IBM i.
How about using this doc page? https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-creating-ssh-key